jupyter_core/troubleshoot.py

```diff
@@ -74,16 +74,20 @@ def main():
     print('\n' + 'platform.platform():')
     print('\t' + environment_data['platform'])
 
-    print('\n' + 'which -a jupyter:')
-    print('\t' + environment_data['which'])
+    if environment_data['which']:
+        print('\n' + 'which -a jupyter:')
```

Comment:
Looks like `where` is already `which -a`, so it's exactly what we want. Yay, Windows!